datalad_next/patches/push_to_export_remote.py

```diff
@@ -0,0 +1,145 @@
+import logging
+from typing import (
+    Dict,
+    Generator,
+    Iterable,
+    Optional,
+    Union,
+)
+from unittest.mock import patch
+
+import datalad.core.distributed.push as push
+from datalad.distribution.dataset import Dataset
+from datalad.runner.exception import CommandError
+from datalad.support.annexrepo import AnnexRepo
+from datalad.support.exceptions import CapturedException
+from datalad_next.credman import CredentialManager
+from datalad_next.utils import (
+    get_specialremote_credential_envpatch,
+    get_specialremote_credential_properties,
+    needs_specialremote_credential_envpatch,
+)
+
+
+__docformat__ = "restructuredtext"
+
+
+lgr = logging.getLogger('datalad_next.push_to_export_remote')
```

Comment:
I think, it is better to use the same logger as push (i.e. same name).
```suggestion
lgr = logging.getLogger('datalad.core.distributed.push')
```

at least that is what is done elsewhere.